ticker/src/deploy.ts

```diff
@@ -93,8 +94,20 @@ const isLatestSlot = (slot1: ISlot, slot2: ISlot) => {
   );
   let fromSlot = nextSlot(deployedSCEvent.context.slot);
   console.log(`listening autonomous SC events on "${addr}"`);
+  let isWatching = process.argv.includes('--watch');
+  let watchingCount = -1;
 
-  while (1) {
+  if (isWatching && process.argv.length > 3) {
+    watchingCount = parseInt(process.argv[3]);
+    console.log(`watching for ${watchingCount} events`);
+  }
+  let watchedCount = 0;
+
+  while (
```

Comment:
The job seems to never stop

Reply:
With what command is it never stopping ? It's supposed to if ènpm run deploy:watch' is used for instance.